packages/tables/resources/views/components/header/description.blade.php

```diff
@@ -1,3 +1,6 @@
-<p {{ $attributes->class(['text-gray-900 filament-tables-header-description']) }}>
+<p {{ $attributes->class(['filament-tables-header-description']) }}>
+    'text-gray-900 filament-tables-header-description',
+    'dark:text-gray-100' => config('filament.dark_mode')
+]) }}>
```

Comment:
Whoops, I just found out how to suggest changes for multiple lines. These lines need removal of course.

```suggestion
```

Reply:
Done! :)